To: vim_dev@googlegroups.com
Subject: Patch 7.3.369
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.369
Problem:    When compiled with Gnome get an error message when using --help.
Solution:   Don't fork. (Ivan Krasilnikov)
Files:	    src/main.c


*** ../vim-7.3.368/src/main.c	2011-10-04 16:43:49.000000000 +0200
--- src/main.c	2011-12-01 20:27:11.000000000 +0100
***************
*** 3294,3300 ****
--- 3294,3303 ----
  #ifdef FEAT_GUI_GNOME
      /* Gnome gives extra messages for --help if we continue, but not for -h. */
      if (gui.starting)
+     {
  	mch_msg("\n");
+ 	gui.dofork = FALSE;
+     }
      else
  #endif
  	mch_exit(0);
*** ../vim-7.3.368/src/version.c	2011-12-08 15:17:28.000000000 +0100
--- src/version.c	2011-12-08 15:57:03.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     369,
  /**/

-- 
Vi is clearly superior to emacs, since "vi" has only two characters
(and two keystrokes), while "emacs" has five.  (Randy C. Ford)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
